web/src/lib/search/cmsCollections.ts

```diff
@@ -77,6 +77,7 @@ export const cmsCollections = [
       "404 Page",
       "Global Error Messages",
       "Navigation Config",
```

Comment:
dust - added a new CMS collection for Component Accessibility, this feels like something we need as some components will have specific accessibility content that won't be pairs with any other content.